src/components/BaseLineChart.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <Line
+    id="my-chart-id"
+    :options="chartOptions"
+    :data="chartData"
+  />
+</template>
+
+<script>
+import { Line } from 'vue-chartjs';
+import { Chart as ChartJS, Title, Tooltip, Legend, PointElement, LineElement, CategoryScale, LinearScale } from 'chart.js';
+
+ChartJS.register(Title, Tooltip, Legend, LineElement, PointElement, CategoryScale, LinearScale);
+
+export default {
+  name: 'BarChart',
+  components: { Line },
+  props: {
+    labels: {
+      type: Array,
+      required: true,
+    },
+    dataSets: {
+      type: Array,
+      required: true,
```

Comment:
Should somehow communicate the types of these. Labels is supposed to be an array of strings for the x-axis labels and datasets matches the [chartjs dataset type](https://www.chartjs.org/docs/latest/general/data-structures.html). This would be a great usecase for typescript, but since that's not configured maybe we should use a vaildator, or even just a comment above the prop. @seriouslysean any thoughts on how to clarify these?